trompace/queries/person.py

```diff
@@ -0,0 +1,67 @@
+# Generate GraphQL queries for mutations pertaining to persons/artists objects.
+from trompace.exceptions import UnsupportedLanguageException, MimeTypeException
+from . import StringConstant
+from .templates import query_create
+from ..constants import SUPPORTED_LANGUAGES
+
+QUERY_PERSON = '''query {{
+  Person(
+  {parameters}
+  )
+  {{
+    identifier
+    name
+    publisher
+    contributor
+    creator
+    source
+    description
+    language
+  }}
+}}'''
+
+QUERY_PERSON_ALL = '''query {
+  Person
+  {
+    identifier
+    name
+    publisher
+    contributor
+    creator
+    source
+    description
+    language
+  }
+}'''
+
+
+def query_artist(identifier: str=None, artist_name=None, publisher=None, contributor=None, creator=None, subject=None,
```

Comment:
let's try and be consistent with terminology here. The object in the CE is a Person, so we should use that word in the client, not artist.

Reply:
That makes sense, I was a but confused myself. But I guess this is what was there in the old guidelines for the API. I'll change.